### DISCUSSION TITLE
Penticost Sunday Ranking

### DISCUSSION THREAD

**Discussion by pvanthony** [romcal/romcal#325] [General]
Should Pentecost Sunday ranking be solemnity?

Version 2 is solemnity
Version 3 is sunday

Should be solemnity correct?

Please advice.

**Comment by emagnier** [comment]
Good catch, the rank & precedence for Pentecost Sunday are indeed incorrect.

https://github.com/romcal/romcal/blob/1327490ea09a9c4e5daa34263074c3cc0c5ef360/lib/general-calendar/proper-of-time.ts#L462-L463

It should be `Precedences.ProperOfTimeSolemnity_2`.

**Issue created:** https://github.com/romcal/romcal/issues/326

**Reply by emagnier** [reply]
@pvanthony thank you for this catch. Also, if you find similar bugs like that, don't hesitate to create directly an issue.

**Reply by pvanthony** [reply]
Will do.

**Reply by pvanthony** [reply]
Did the change as mentioned in https://github.com/romcal/romcal/discussions/325#discussioncomment-1630522 compiled it and it works great. Thank you for the solution.
